telethon/client/chats.py

```diff
@@ -873,6 +873,12 @@ def action(
         )
 
         if isinstance(entity, types.InputPeerChannel):
+            if post_messages is None or edit_messages is None:
```

Comment:
"If it's `None` and it's a `megagroup`, set to `None`". Surely you meant to do the following instead?:
```suggestion
            if post_messages or edit_messages:
```